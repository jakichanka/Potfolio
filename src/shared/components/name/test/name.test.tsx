/**
 * @jest-environment jsdom
 */

import { MyName } from ".."
import renderer from "react-test-renderer"

it("testing component with snapshot", () => {
  const component = renderer.create(<MyName />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
