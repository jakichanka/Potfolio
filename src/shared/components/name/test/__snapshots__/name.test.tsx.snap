// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing component with snapshot 1`] = `
<div
  className="chakra-fade"
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <h2
    className="chakra-heading css-6kld57"
  />
</div>
`;
